src/metabase/email/pulse.mustache

```diff
@@ -1,17 +1,123 @@
-{{> metabase/email/_header}}
-  {{#pulseName}}
-    {{#pulseLink}}
-      <a href="{{pulseLink}}" class="title-link">
-        <h1 style="{{sectionStyle}} margin: 24px 0 24px 0;">
-          {{pulseName}}
-        </h1>
-      </a>
-    {{/pulseLink}}
-    {{^pulseLink}}
-      <h1 style="{{sectionStyle}} margin: 16px; color: {{color-brand}};">
-        {{pulseName}}
-      </h1>
-    {{/pulseLink}}
-  {{/pulseName}}
-  {{{pulse}}}
-{{> metabase/email/_footer_pulse}}
+<html>
```

Comment:
I think you'll want to add a `<!doctype html>` as the first line of the template. I'm not familiar with what email clients do with html, but it'll help prevent it from rendering in quirks mode https://developer.mozilla.org/en-US/docs/Glossary/Doctype